MovieFlix/app/build.gradle

```diff
@@ -0,0 +1,62 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.example.movieflix'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.example.movieflix"
+        minSdk 24
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    buildFeatures {
+        dataBinding true
+
+    }
+}
+
+dependencies {
+    implementation 'com.squareup.picasso:picasso:2.71828'
+    implementation 'androidx.room:room-runtime:2.5.2'
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0 (retrofit-2.9.0.jar)'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.1'
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4"
```

Comment:
there should be consistency in your formatting... if you are using single quotation marks then use that format for all the dependencies